web/beacon-app/cypress/e2e/OrganizationDashboard/OrganizationDashboard.ts

```diff
@@ -0,0 +1,46 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+beforeEach(() => {
+  cy.loginWith({ email: 'mpie6802@gmail.com', password: 'elyseebleu1@gmail.com1A' });
```

Comment:
we will need to create a story to be sure to use env data for the user credentials.